backend/model/model.go

```diff
@@ -2000,10 +2000,14 @@ func (obj *SessionAlert) SendAlerts(ctx context.Context, db *gorm.DB, mailClient
 
 type Service struct {
 	Model
-	ProjectID      int                       `gorm:"not null;uniqueIndex:idx_project_id_name"`
-	Name           string                    `gorm:"not null;uniqueIndex:idx_project_id_name"`
-	Status         modelInputs.ServiceStatus `gorm:"not null;default:created"`
-	GithubRepoPath *string
+	ProjectID       int                       `gorm:"not null;uniqueIndex:idx_project_id_name"`
+	Name            string                    `gorm:"not null;uniqueIndex:idx_project_id_name"`
+	Status          modelInputs.ServiceStatus `gorm:"not null;default:created"`
+	GithubRepoPath  *string
+	BuildPrefix     *string
+	GithubPrefix    *string
+	LastSeenVersion *string
+	ErrorDetails    *string
```

Comment:
Can this be an array of strings? That way we can handle multiple reasons for their being an issue.

Reply:
Just pushed up these changes in https://github.com/highlight/highlight/pull/6283/commits/1719129587f426583f5888ef1753b3a457bb2fec

Reply:
Awesome!